### DISCUSSION TITLE
GitHub Deleted My Account and Continues to Ignore Support Requests

### DISCUSSION THREAD

**Discussion by betebetoven2** [community/community#145169] [GitHub Education]
Hi everyone,

I’m reaching out because I’ve been facing an incredibly frustrating situation with GitHub for almost a month now. My student account was deleted because GitHub couldn’t verify my status as a student. However, I am a student and I can provide the necessary documentation.

Since then, I’ve submitted multiple tickets and emails from three different accounts, but I’ve received no response. To try and move forward, I even created a new GitHub account and subscribed to the $4/month plan, yet my requests are still being ignored.

Has anyone experienced a similar issue? If so, how were you able to resolve it?

At this point, I feel like I have no other option but to bring attention to this matter here in the community. Any advice or guidance would be greatly appreciated.

Thank you in advance!

**Comment by ujamX** [comment]
I also remember deleting my account and getting it restored.
I remember creating several tickets because there was no response.
The method that was effective for me was to write a post every day to share the current progress in a ticket I had already created.
That's how I got a reply and the deleted account was restored according to the procedure.
I wish you good results
